src/components/accordion/Accordion.spec.jsx

```diff
@@ -178,9 +178,9 @@ describe('<Accordion>', () => {
       </Accordion>
     );
 
-    expect(accordion.find('[aria-labelledby]').hostNodes()).toHaveLength(
-      accordionIds.length
-    );
+    expect(
```

Comment:
I would keep changes only related to the PR and prepare separate pr to improve other component

Reply:
Adding `aria-labelledby` in `Icon` caused this test to fail, without this change "Continuos Integration / Build and Test (push)" step fails.